Assets/Gothic-UnZENity-Core/Scripts/GameManager.cs

```diff
@@ -0,0 +1,185 @@
+		private BarrierManager _barrierManager;
+		private SkyManager _skyVisualManager;
+		private StationaryLightsManager _stationaryLightsManager;
+		private XRDeviceSimulatorManager _xrSimulatorManager;
+		private GameTime _gameTimeManager;
+		private MusicManager _gameMusicManager;
+		private GUZSceneManager _gameSceneManager;
+		private LoadingManager _gameLoadingManager;
+		private RoutineManager _npcRoutineManager;
+		private FileLoggingHandler _fileLoggingHandler;
+		private TextureManager _textureManager;
+		private FontManager _fontManager;
+
+		private GameSettings _settings;
+		private bool _isInitialised = false;
+
+		public GameSettings Settings => _settings;
+		public GameConfiguration Config => config;
+		public SkyManager Sky => _skyVisualManager;
+		public GameTime Time => _gameTimeManager;
+		public RoutineManager Routines => _npcRoutineManager;
+		public TextureManager Textures => _textureManager;
+		public GUZSceneManager Scene => _gameSceneManager;
+		public FontManager Font => _fontManager;
+        public StationaryLightsManager Lights => _stationaryLightsManager;
+        public VobMeshCullingManager MeshCulling => _meshCullingManager;
+        public VobSoundCullingManager SoundCulling => _soundCullingManager;
+
+		// ReSharper disable Unity.PerformanceAnalysis
+		private void Load()
+		{
+			// If the Gothic installation directory is not set, show an error message and exit.
+			if (!_settings.CheckIfGothic1InstallationExists())
+			{
+				invalidInstallationPathMessage.SetActive(true);
+				return;
+			}
+
+			// Otherwise, continue loading Gothic.
+            ResourceLoader.Init(_settings.GothicIPath);
+	
+            _gameMusicManager.Init();
+            
+			GUZBootstrapper.BootGothicUnZENity(config, _settings.GothicIPath, _settings.GothicILanguage);
+			_gameSceneManager.LoadStartupScenes();
+
+			if (config.enableBarrierVisual)
+			{
+				GlobalEventDispatcher.WorldSceneLoaded.AddListener(() =>
+				{
+					_barrierManager.CreateBarrier();
+				});
+			}
+		}
+
+		private void Awake()
+		{
+			GameGlobals.Instance = this;
+			LookupCache.Init();
+
+			_textureManager = GetComponent<TextureManager>();
+			_fontManager = GetComponent<FontManager>();
+			_settings = GameSettings.Load();
+			_fileLoggingHandler = new FileLoggingHandler(_settings);
+			_gameLoadingManager = new LoadingManager();
+			_meshCullingManager = new VobMeshCullingManager(config, this);
+			_soundCullingManager = new VobSoundCullingManager(config);
+			_barrierManager = new BarrierManager(config);
+			_stationaryLightsManager = new StationaryLightsManager();
+			_xrSimulatorManager = new XRDeviceSimulatorManager(config);
+			_gameTimeManager = new GameTime(config, this);
+			_skyVisualManager = new SkyManager(config, _gameTimeManager, _settings);
+			_gameMusicManager = new MusicManager(config);
+			_gameSceneManager = new GUZSceneManager(config, _gameLoadingManager, xrInteractionManager);
+			_npcRoutineManager = new RoutineManager(config);
```

Comment:
That's the _new()_ behaviour of Managers I also asked in general. What's the benefit of it?